### PR TITLE
Focus Controller Testing

### PR DESCRIPTION
**Pull request by crestu**
### Changes
Testing the index action in the Focuses Controller.

### Link to Issue 
https://app.asana.com/0/1206867754300740/1207175839749228

### Before you submit for review, Did you...

- [X] Added relevant tests (if applicable)
- [X] Linked Asana ticket to PR
- [X] Added comments, notes, or questions for reviewwer
- [X] Added relevant screenshots
- [X] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

**Review by ridwvn22 — DISMISSED** [review]
looks clean

**Review by fouziiah — COMMENTED** [review]
Test passed for me, looks great!

File: test/controllers/focus_controller_test.rb [hunk 1]
```diff
@@ -2,8 +2,22 @@
 
 require 'test_helper'
 
-class FocusControllerTest < ActionDispatch::IntegrationTest
-  # test "the truth" do
-  #   assert true
-  # end
+class FocusControllerTest < ActionController::TestCase
```
Comment (edithemmings) [inline test/controllers/focus_controller_test.rb:5]: What was the reason for changing the base class to `ActionDispatch::IntegrationTest`?

File: test/controllers/focus_controller_test.rb [hunk 2]
```diff
@@ -2,8 +2,22 @@
...
+  end
+
+  test 'should get index' do
+    student = build(:student)
+    create(:student_focus, student:)
+    get :index, params: { student_id: student.id }
+    assert_response :success
+  end
+
+  test 'should not save focus without a name' do
```
Comment (edithemmings) [inline test/controllers/focus_controller_test.rb:19]: This doesn't appear to be testing the controller. Might be more relevant to the model test?

**Review by edithemmings — APPROVED** [review]
🚀

**Review by KNAthari — APPROVED** [review]

**Pull request merged**